listenbrainz/domain/lastfm.py

```diff
@@ -0,0 +1,119 @@
+import uuid
+
+import requests
+from flask import current_app
+from psycopg2.extras import execute_values
+from requests.adapters import HTTPAdapter, Retry
+from sqlalchemy import text
+
+from listenbrainz import db
+from brainzutils import musicbrainz_db
+
+from listenbrainz.webserver.errors import APINotFound
+
+
+def bulk_insert_loved_tracks(user_id: int, feedback: list[tuple[int, str]]):
+    # DO NOTHING because we can only import loved tracks.
+    # if a user has imported something before from last.fm then the score doesn't change.
+    # if the user has unloved a track on last.fm, we won't receive it this time so can't
```

Comment:
I have a feeling that this case doesn't happen all that often.

Reply:
Yes probable. If someone complains, we can check if anything can be done.